bench/src/benches/confirmation_rate.rs

```diff
@@ -31,29 +30,25 @@ pub struct RpcStat {
 }
 
 /// TC2 send multiple runs of num_txns, measure the confirmation rate
-#[tokio::main]
-async fn main() -> anyhow::Result<()> {
-    tracing_subscriber::fmt::init();
-
+pub async fn confirmation_rate(
+    payer_path: String,
```

Comment:
nit: I'd prefer strong typing where possible. i.e. Path here

PathBuf vs Path: Path only holds a reference to the path string data but doesn't own this data, while PathBuf owns the string data itself